cclib/bridge/cclib2pyscf.py

```diff
@@ -8,6 +8,11 @@
 """Bridge for using cclib data in PySCF (https://github.com/pyscf/pyscf)."""
 
 from cclib.parser.utils import find_package
+import qcelemental as qe

```

Comment:
Does this require QCElemental? IIRC we haven't started depending on it yet. I think we will, not sure if there was a plan around that. @berquist ? @shivupa

Reply:
Ah yes, This was a question I had too. I saw a few pull request discussing it. I can change this to use the internal conversion from cclib.parser.utils in the mean time though if that is preffered.

Reply:
if we are planning to use QCElemental anyways (this was my impression) I am fine with it coming in on this PR

Reply:
I stashed it away when separating out https://github.com/cclib/cclib/pull/972 from https://github.com/cclib/cclib/pull/781. Since this PR doesn't technically depend on on qcel, can you change it to use our built-in class?

Reply:
Ah, I see. I have replaced the qcelemental call with the built-in PeriodicTable class.